Psychtoolbox/PsychProbability/Shuffle.m

```diff
@@ -11,18 +11,78 @@
 % xx/xx/92  dhb  Wrote it.
 % 10/25/93  dhb  Return index.
 % 5/25/96   dgp  Made consistent with sort and "for i=Shuffle(1:10)"
-% 6/29/96	  dgp  Edited comments above.
+% 6/29/96	dgp  Edited comments above.
 % 5/18/02   dhb  Modified code to do what comments say, for matrices.
 % 6/2/02    dhb  Fixed bug introduced 5/18.
 
-[null,index] = sort(rand(size(X)));
-[n,m] = size(X);
-Y = zeros(size(X));
-if n == 1 || m == 1
-	Y = X(index);
-else
-	for j = 1:m
-		Y(:,j)  = X(index(:,j),j);
-	end
+% 9/10/15   niki Add bindDim. so that this function can shuffle with the
```

Comment:
Thanks for your comments, but I don't know how to create a help text. Should I put them into a separate file or put them into the script with some specific formats?